packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -229,6 +238,40 @@ export interface SetLoanToken {
   interest?: BigNumber
 }
 
+export interface ListLoanTokenData {
+  [key: string]: ListLoanTokenDetail
+}
+
+export interface ListLoanTokenDetail {
+  token: TokenData
+  priceFeedId: string
+  interest: BigNumber
+}
+
+export interface TokenData {
+  [key: string]: TokenDetail
+}
+
+export interface TokenDetail {
```

Comment:
```suggestion
export interface LoanTokenInfo {
```

Reply:
1. follow our naming standard here
`result { info }`

2. prefix the `Loan` for better differentiate them if we don't refer to `Token` interface